.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "build_manpages"]
+	path = build_manpages
+	url = https://github.com/praiskup/build_manpages
```

Comment:
I wish it was a dedicated python package.

Reply:
I don't plan to work on separate package now;  at least until we stabilize the argparse -> manpage convertor :).  Any other life-cycle than git-submodule would be to slow at this point.

Reply:
I didn't meant to ask you to do the separate package, I was just sad with the distribution model. I'm glad it works for you as it is.

Reply:
And I'm not against wrapping it as a real python package. +1 to any attempt towards this.